nextflow/data_processing_pipeline.nf

```diff
@@ -0,0 +1,50 @@
+nextflow.preview.dsl = 2
```

Comment:
It's probably not a good idea to have something named 'data_processing_pipeline.py' and 'data_processing_pipeline.nf' in the same repo

Reply:
I think it will be fine. To an extent, this should eventually replace it or at least parts of it. That or you can think of a better name for me